.github/workflows/convert.yml

```diff
@@ -0,0 +1,56 @@
+name: convert precommit and test
+run-name: convert precommit and test
+on:
+  pull_request:
+    paths:
+      - convert/**
+      - .github/workflows/convert.yml
+jobs:
+  convert-pre-commit-actions:
+    strategy:
+      matrix:
+        python-version: [ "3.8.15" ]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: "pip"
+      - name: Run isort
+        uses: isort/isort-action@v1.1.0
+        with:
+          sort-paths: convert/*
+          configuration:
+            --check-only
+            --profile=black
+            --line-length=79
+            --skip=tests
+      - name: Black
+        uses: psf/black@stable
+        with:
+          options: "--line-length=79"
+      - run: pip install flake8
+      - run: flake8 --extend-ignore=E203 convert
```

Comment:
'alembic' and 'tests' should be escaped. See relevant commands in .github/workflows/jobs.yaml